packages/dd-trace/src/format.js

```diff
@@ -69,6 +70,10 @@ function extractTags (trace, span) {
     addTag({}, trace.metrics, MEASURED, 1)
   }
 
+  if (!tags['span.kind'] || !(tags['span.kind'] in ['client', 'producer'])) {
```

Comment:
Actually, why was this changed in the first place? My understanding is that the requirement is that the tag is present on all spans for the service, so checking based on the service name would be the best way to ensure of that, especially if the service name is changed manually.

Reply:
The requirement for setting language is now to set the language tag for every case except one. That one case is when span.kind is either 'client' or 'producer' as these represent operations within services external to the language. Will reach out on slack with a location detailing the new intended specification.

Reply:
What if a child span of a client span has the same service name as the process?

Reply:
Can you please provide a bit more detail of what you mean? I haven't seen a case where this could be a problem